sklearn/plot/_gridsearch_results.py

```diff
@@ -0,0 +1,116 @@
+import numpy as np
+from sklearn.plot import plot_heatmap
+
+
+def plot_gridsearch_results(cv_results, metric='mean_test_score',
```

Comment:
i meant its less code with GridSearchCV and the plotting function. We can do what validation example does with following code:

```
import matplotlib.pyplot as plt
import numpy as np

from sklearn.datasets import load_digits
from sklearn.svm import SVC
from sklearn.model_selection import GridSearchCV
from sklearn.plot import plot_gridsearch_results

digits = load_digits()
X, y = digits.data, digits.target

param_grid = {'gamma':np.logspace(-6, -1, 5)}
gs = GridSearchCV(
    SVC(), param_grid=param_grid,
    cv=10, scoring="accuracy", n_jobs=-1)

gs.fit(X, y)
plot_gridsearch_results(gs.cv_results_)
```